app/components/info-card.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { cloneElement, FC, isValidElement } from 'react';
+import { Box, BoxProps, color, Flex, FlexProps, Stack, StackProps, Text } from '@stacks/ui';
+import { ExplainerTooltip } from '@components/tooltip';
+import { Hr } from '@components/hr';
+
+export const InfoCard: FC<FlexProps> = props => (
+  <Flex
+    flexDirection="column"
+    boxShadow="low"
+    border={`1px solid ${color('border')}`}
+    borderRadius="8px"
+    minHeight="84px"
+    {...props}
+  />
+);
+
+export const InfoCardGroup: FC<BoxProps> = ({ children, ...props }) => {
+  const parsedChildren = Array.isArray(children) ? children : [children];
+  const infoGroup = parsedChildren.flatMap((child, index) => {
+    if (!isValidElement(child)) return null;
+    return [
+      cloneElement(child, {
+        key: index,
+        mb: index === parsedChildren.length ? '280px' : null,
+      }),
+      index !== parsedChildren.length - 1 && <Hr my="loose" key={index.toString() + '-hr'} />,
+    ];
+  });
+  return <Box {...props}>{infoGroup}</Box>;
+};
+
+export const InfoCardSection: FC<StackProps> = ({ children, ...props }) => (
+  <Stack {...props} spacing="base-tight">
+    {children}
+  </Stack>
+);
+
+export const InfoCardRow: FC<FlexProps> = props => (
+  <Flex justifyContent="space-between" {...props} />
+);
+
+interface InfoCardLabelProps extends FlexProps {
```

Comment:
This is my first time working with interfaces, is it typical just to throw them in above where they are used in a file or migrate them to the top? Any standard preferred? I assume bc of where it is here, we just place them where needed?

Reply:
for me at least, if there is just one or two things in a given component/file, I'll put the interface above the component. If it's more complex or has many types/interfaces, I'll opt for a types.ts file for the component

Reply:
In this instance, as it's the `interface` that types the props this component should have, yes. Makes sense to put them in with the component. If there are reusable ones, however, they'd belong somewhere else; such as the `models` directory

Reply:
Got it, thanks!